lxd/project/permissions.go

```diff
@@ -154,14 +156,18 @@ func checkRestrictions(project *api.Project, instances []db.Instance) error {
 				}
 				return nil
 			}
+		case "restricted.containers.lowlevel":
+			if restrictionValue == "allow" {
+				allowContainerLowLevel = true
+			}
 		}
 	}
 
 	for _, instance := range instances {
 		if instance.Type == instancetype.Container {
 			for key, value := range instance.Config {
 				// First check if the key is a forbidden low-level one.
-				if isContainerLowLevelOptionForbidden(key) {
+				if !allowContainerLowLevel && isContainerLowLevelOptionForbidden(key) {
```

Comment:
Ok, so that's one step in the right direction, hopefully later commits get us all the way there ;)

Reply:
Yup :)